elements/pl_number_input/pl_number_input.py

```diff
@@ -51,7 +51,9 @@ def render(element_html, element_index, data):
             info = chevron.render(f, info_params).strip()
         with open('pl_number_input.mustache', 'r', encoding='utf-8') as f:
             info_params.pop('format', None)
-            info_params['shortformat'] = True
+            # Within mustache, the shortformat generates the shortinfo that is used as a placeholder inside of the numeric entry.
+            # Here we opt to not generate the value, hence the placeholder is empty.
+            info_params['shortformat'] = not pl.get_boolean_attrib(element, 'hide_help_text', False)
```

Comment:
Any reason you don't just immediately default to the empty string if help text is disabled? Seems wasteful to go through the template for that. In fact, you could probably skip `# Get comparison parameters and info strings` and down if help text is disabled, since `info` won't be used either.

Reply:
Yes, I think that is definitely accurate. However, I'm picking up a weird error on the example problem after rebasing. Any thoughts as to why the test values are `null`?

<img width="883" alt="screen shot 2018-05-30 at 11 13 43 pm" src="https://user-images.githubusercontent.com/833642/40761060-6c3760f8-645f-11e8-85b6-26f7267383a9.png">
<img width="1122" alt="screen shot 2018-05-30 at 11 00 33 pm" src="https://user-images.githubusercontent.com/833642/40761062-6e80f54a-645f-11e8-996b-9cc254749ed0.png">